src/data/fintype.lean

```diff
@@ -161,21 +161,24 @@ theorem card_congr {α β} [fintype α] [fintype β] (f : α ≃ β) : card α =
 by rw ← of_equiv_card f; congr
 
 theorem card_eq {α β} [F : fintype α] [G : fintype β] : card α = card β ↔ nonempty (α ≃ β) :=
-⟨λ h, ⟨by classical; 
+⟨λ h, ⟨by classical;
   calc α ≃ fin (card α) : trunc.out (equiv_fin α)
      ... ≃ fin (card β) : by rw h
-     ... ≃ β : (trunc.out (equiv_fin β)).symm⟩, 
+     ... ≃ β : (trunc.out (equiv_fin β)).symm⟩,
 λ ⟨f⟩, card_congr f⟩
 
 def of_subsingleton (a : α) [subsingleton α] : fintype α :=
 ⟨finset.singleton a, λ b, finset.mem_singleton.2 (subsingleton.elim _ _)⟩
 
-@[simp] theorem fintype.univ_of_subsingleton (a : α) [subsingleton α] :
+@[simp] theorem univ_of_subsingleton (a : α) [subsingleton α] :
   @univ _ (of_subsingleton a) = finset.singleton a := rfl
 
-@[simp] theorem fintype.card_of_subsingleton (a : α) [subsingleton α] :
+@[simp] theorem card_of_subsingleton (a : α) [subsingleton α] :
   @fintype.card _ (of_subsingleton a) = 1 := rfl
 
+lemma fintype.card_eq_sum_ones {α} [fintype α] : fintype.card α = (fintype.elems α).sum (λ _, 1) :=
```

Comment:
This `fintype.` is also redundant.

Reply:
Also use `univ` instead of `fintype.elems`